proto/timeseries.proto

```diff
@@ -46,6 +46,22 @@ message TimeSeriesData {
   repeated TimeSeriesDatapoint datapoints = 3;
 }
 
+// TimeSeriesQueryAggregator describes a set of aggregation functions which are
+// applied to data points before returning them as part of a query. 
+//
+// Cockroach does not store data points at full fidelity, instead "downsampling"
+// data points into fixed-length sample periods. The value returned for each
+// sample period is equivalent to applying the supplied aggregator function to
+// the original data points that fell within the sample period.
+enum TimeSeriesQueryAggregator {
+  // AVG returns the average value of points within the sample period.
+  AVG = 1;
+  // AVG_RATE returns the differential of the average for the sample period.
```

Comment:
it's arguably a correct term but it means so many things. I'd go for "rate of change".